README.rst

```diff
@@ -18,7 +18,11 @@ The intended usage is to create a ``buildout.cfg`` like::
     extends = https://raw.githubusercontent.com/collective/buildout.plonetest/master/test-5.x.cfg
     package-name = plone.app.foo
 
-Create a virtualenv and run buildout.
+Create a virtualenv and run buildout (mind the `-N`)::
+
+    python -m venv
+    bin/buildout -N
```

Comment:
Some considerations:

- The documentation has been updated to focus on Plone 5. In the Plone 5 buildout we have [newest = false](https://github.com/collective/buildout.plonetest/blob/3b03ec8813f62ce844a6ae21544dbb26b6c17b92/plone-5.2.x.cfg#L16). This does the same thing as run the buildout with `-N`. So I think `-N` is reduntande here.

- Using `bin/buildout -N` or `newest = false` makes that unpinned packages in buildout are not updated when we run the buildout, even if new versions are already available. I think this is bad for CI. I've seen several incompatibility errors with new versions of dependencies that could have been discovered before, if `newest = false` was not used. I really don't like it and I'm thinking about doing a PR to remove the `newest = false`.

- When using `bin/buildout -N` or` newest= false` the `setuptools/buildout` versions pinned in cfg are ignored. Both `bin/instance` and `bin/test` will use the versions in your Python's site-packages and ignore cfg versions.

- Using `bin/buildout -N` or` newest= false` can cause problems with #60.

- To avoid restart problems in the buildout, just use `pip install -r requirements.txt`, with `requirements.txt` containing the same versions of `setuptools/buildout` pinned in cfg. I explained it [here](https://github.com/collective/buildout.plonetest/issues/60#issuecomment-561157545).

Reply:
As written in https://github.com/collective/buildout.plonetest/issues/53#issuecomment-582431526 I'm giving up on this - it is a moving target, not reliablly reproducible even just fourmonths later.

Nevertheless some remarks:

* `newest= false` was added for exactly the same reason as  #53. for Plone 4: Avoid endless loops, see ee3755090d89c833fbd.
* `newest= false` is only in the Plone 5 variants, not in the older ones. Thus the `-N` is not redundant.

> … unpinned packages in buildout are not updated when we run the buildout, even if new versions are already available. I think this is bad for CI. …

1. The Readme and the repo are not only about CI, but also about local testing.
2. Whether this is good or bad depends on the aims of your CI and what you want to trst against.